search/src/test/scala/weco/api/search/services/AggregationsTest.scala

```diff
@@ -117,49 +92,39 @@ class AggregationsTest
         whenReady(aggregationQuery(index, searchOptions)) { aggs =>
           aggs.format should not be empty
           val buckets = aggs.format.get.buckets
-          buckets.length shouldBe formats.length
-          buckets.map(_.data) should contain theSameElementsAs formats
+          buckets.length shouldBe works.length
```

Comment:
Is the bucket content checked elsewhere now?  (Same question relates to line 113, below).

Reply:
No, oops. I took it out to get the test working and forgot to put it back, will fix. 🙈